src/RemixDownloader/RemixDownloader.Console/Program.cs

```diff
@@ -149,9 +149,11 @@ private static async Task DownloadAllFilesAsync(IEnumerable<ModelResult> items,
 
                     // *** Phase 1 - Always downloading the original model file *** //
 
-                    // Create a subfolder for each group of files.
-                    var modelSubfolder = selectedFolder.CreateSubdirectory(item.Name);
 
+                    // Create a subfolder for each group of files.
+                    // The ID is suffixed to prevent collisions of models with the same name
+                    var modelSubfolder = selectedFolder.CreateSubdirectory($"{item.Name}-{item.Id}");
```

Comment:
Models are allowed to have identical names. The Microsoft account has multiple models named 'Bursting Fireworks'

Reply:
I'll run some tests again checking for any skipped/zerobyte files. The tests I ran so far weren't ont he quality of the saved data but rather that there were no conflicts that crashed the process.